script.js

```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+function setFilter(filterValue) {
+  const filterRadioButton = document.getElementById(
+    `option${filterValue === "all" ? 1 : filterValue === "completed" ? 3 : 2}`
+  );
+  if (filterRadioButton) {
+    filterRadioButton.checked = true;
+    state.filter = filterValue;
+    renderTodos();
+  } else {
+    console.error("Invalid filter value provided.");
+  }
+}
+
+setFilter("all");
```

Comment:
Apparently you have called the function once. I believe you won't need to call it more than once later on as well. Unless you plan to save the filter state in between browser sessions, so you will need to set the filter from saved state. But I think it isn't needed as user might be happy with initial value 'all' each time the app gets reloaded, right?
What am I heading to? You do not need javascript here at all just set `checked` radio button in html and the filter state is already initially set to 'all'.